lib/ranked-model.rb

```diff
@@ -65,6 +65,16 @@ def ranks *args
         ranker.with(self).relative_rank
       end
 
+      define_method "#{ranker.name}_insert_after!" do |previous_record|
```

Comment:
I think we should maybe change the method name to `_rank_after!` as `insert` implies a new record.

Reply:
Yes, definitely a better name 👍